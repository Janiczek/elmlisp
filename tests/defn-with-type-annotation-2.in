(defn foo : (-> Int (List a))
  (bar)
  "lying!")
