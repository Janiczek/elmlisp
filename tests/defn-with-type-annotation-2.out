foo : Int -> List a
foo bar =
    "lying!"
